src/main/scala/org/apache/spark/streaming/amqp/AMQPRateController.scala

```diff
@@ -22,12 +22,12 @@ import java.util.concurrent.{Executors, ScheduledExecutorService, ScheduledFutur
 
 import com.google.common.util.concurrent.{RateLimiter => GuavaRateLimiter}
 import io.vertx.proton.{ProtonDelivery, ProtonMessageHandler, ProtonReceiver}
-import org.apache.qpid.proton.amqp.{Symbol => AmqpSymbol}
 import org.apache.qpid.proton.amqp.messaging.Rejected
 import org.apache.qpid.proton.amqp.transport.ErrorCondition
+import org.apache.qpid.proton.amqp.{Symbol => AmqpSymbol}
```

Comment:
Why this line changed its position. Just the IDE ? :-) Genuine question just to avoid useless changes.

Reply:
ha! I didn't notice this one, yeah, it looks like some IDE magic. I'll remove it, I don't like it